docs/creating_rules.md

```diff
@@ -57,6 +58,13 @@ checks whitespace usage in a document, you can add the `:whitespace` tag, and
 users who don't care about whitespace can exclude that tag on the command line
 or in style files.
 
+You can optionally provide a URL with more context on this rule and why it
+exists. `markdownlint` links to this URL when this rule fails, which can help
+the people receiving a failure understand how to fix it. Docs URLs can also be
+specified once for many rules in a ruleset, the built-in rules
+[use this feature][1] to set a URL dynamically using the rule id and/or
```

Comment:
What if you do the `[use this feature](<url>)` syntax? Then you can do something like:

```
[use this feature](
long_url
) other stuff here
```

maybe? Because we'll ignore single-word long-lines